BuildScripts/AppVeyor-Build.ps1

```diff
@@ -127,12 +127,12 @@ function ExecuteTests($cover)
     {
         cinst opencover.portable -y
         cinst coveralls.io -source https://nuget.org/api/v2/
-        & C:\ProgramData\chocolatey\lib\opencover.portable\tools\OpenCover.Console.exe -register:user -filter:"+[Pretzel.Logic]*" -excludebyattribute:*.ExcludeFromCodeCoverage* -target:"%xunit20%\xunit.console.exe" -targetargs:"""src\Pretzel.Tests\bin\Release\Pretzel.Tests.dll"" -noshadow -appveyor" -output:$artifacts\coverage.xml -returntargetcode
+        & C:\ProgramData\chocolatey\lib\opencover.portable\tools\OpenCover.Console.exe -register:user -filter:"+[Pretzel.Logic]*" -excludebyattribute:*.ExcludeFromCodeCoverage* -target:"$tools\xunit.runner.console\tools\net472\xunit.console.exe" -targetargs:"""src\Pretzel.Tests\bin\Release\Pretzel.Tests.dll"" -noshadow -appveyor" -output:$artifacts\coverage.xml -returntargetcode
         & coveralls.net --opencover $artifacts\coverage.xml
     }
     Else
     {
-        &$tools\xunit\xunit.console.exe "$src\Pretzel.Tests\bin\Release\Pretzel.Tests.dll"
+        &$tools\xunit.runner.console\tools\net472\xunit.console.exe "$src\Pretzel.Tests\bin\Release\Pretzel.Tests.dll"
```

Comment:
I like the idea to get this dependency directly from nuget, but there are other things to consider:
 - it is used in more places (like build.cmd, I know it is a mess, it should be addressed sometime)
 - other dependencies who fits like GitVersion should be downloaded the same way

Reply:
Nevermind, since GitVersion is needed only on AppVeyor it could still have his special treatment.

Reply:
My focus was getting the build to run, so I didn't check the other places it is used so far.